### ISSUE TITLE
Superstructure IDs from Cambridge Ms

### ISSUE THREAD

**Issue by bleierr** [chigc/data#6]
We have discussed to use the wikidata names for Superstructure IDs (e.g. n-keturah) and in some situations (?) a reading that is closer to the manuscripts. I will create now the superstructure IDs and I think that it would be more efficient to use the transcription of the Cambridge version as a starting point. Agnese is already very far with it and it would be easier to just use these names for the IDs. Problem or more a question: have we already used the superstructure IDs somewhere and how much work is it to change them? How do we form superstructure IDs for things that are not in the Cambridge manuscript?

**Comment by Elysir** [comment]
I have already used the superstructure IDs in my NavGraph but I can still change them (it won't be so much work).

**Comment by PatrickSahle** [comment]
yeah, think so as well: right now, we are still save as regards changes<br/>
I'd like to repeat: <br/>
We are totally free with our internal IDs. Still I'd like to see a sensible and functioning "rule" in place. I feel rather uncomfortable with "we go with a totally arbitrary ms" (you point at one of the resulting problems) - but I will swallow that and remain silent ...
We are also totally free with our labels that will be publicly visible. Bonus: as no other system depends upon this, we can decide about these things later

**Comment by Elysir** [comment]
@bleierr Which document should I use as a reference for the IDs? Are you still writing them here https://docs.google.com/spreadsheets/d/1Wg4BcFz5JBXwpQptByYl9abQc_aFmau9AvVpn5etUbE/edit#gid=0 or elsewhere?

**Comment by bleierr** [comment]
@Elysir yes, this is the right document. I will continue with the Superstructure IDs, but wanted to see how this discussion turns out first. 
@PatrickSahle I think the Wikidata naming idea is also not great. Several problems: 
1. many people do not have a wikidata entry and we (or I) have to come up with a name
2. in the case where the Wikidata name is very different from the name in the manuscripts, we decided to use a name closer to a manuscript reading, but what name?
if we go with "a random" manuscript (I don't think Cambridge is totally random) at least we will have readings that are close to what appears in the manuscripts and we have names for all people on the main lines and just have to worry about exceptions. In Wikidata people that are certainly in all relevant manuscripts (because they are part of the line of Christ) often don't have an entry and i would have to invent a name.

**Comment by PatrickSahle** [comment]
This future scenario has to be kept in mind. On a conference or in an argument about an article:
- so how did you build your PIDs?
- we followed manuscript X (and Ms Y and Z in certain cases)
- so in your project you follow the "Leithandschriftenprinzip"?
- well, no ...
- ????
@bleierr @vranzvischer @andrea ( ups, not on gitlab?)

**Comment by bleierr** [comment]
Thanks @amacchiarelli I have seen the transcription of the nodes this morning. Even if Crassus is missing, the manuscript provides most of the nodes and names. We could add "Crassus" from the next manuscript once we start transcribing it.

I think the logic is sound: we use the name as it appears in the first manuscript we transcribe. Cambridge might not be the "Leithandschrift", but it is in the list of the manuscrpts selected by the team. When we find something new we add the name/node. 

The use of wikidata-names brings the same problems, and it does not solve questions such as: how do we name things that are not in wikidata? ...and how do we form the name of something that has a "strange name" in wikidata (Aclima = Calmana)?

If we don't need the Superstructure IDs soon and we have time to figure out what the "Leithandschrift" and "correct" readings of all names is or discuss and clarify all the "problem names" on wikidata. We can do this

Buy the way. I would love that someone asks me at a conference if we formed our PIDs following the "Leithandschriftenprinzip". I am not sure I would answer or just role my eyes :)
We are talking about PIDs here and not about textual criticism. We can also go with n1, n2, n3, n4, but as it is easier if we can read our PIDs we use proper names as part of the PID. But it is still only a string

**Comment by bleierr** [comment]
> This future scenario has to be kept in mind. On a conference or in an argument about an article:
>     * so how did you build your PIDs?
>     * we followed manuscript X (and Ms Y and Z in certain cases)
>     * so in your project you follow the "Leithandschriftenprinzip"?
>     * well, no ...
>     * ????
To get the discussion going again here another suggestion. As it may be difficult so early in the project to know which manuscript has the best reading to form superstructure IDs. Maybe we could use the oldest manuscript? Age could be a suitable argument, or? Other ideas?
